Sources/Client/Client/Client.swift

```diff
@@ -229,3 +244,17 @@ extension Client {
         }
     }
 }
+
+public struct Subscription {
```

Comment:
```suggestion
public final class Subscription {
```

Reply:
Why should it be a class?

Reply:
As I understand it was the idea to store references to make possible to cancel them, but with structs you will store copies.

Reply:
On another hand you should think about thread safety.